models/experimental/llama/tests/test_llama_mlp.py

```diff
@@ -97,7 +97,7 @@ def run_test_LlamaMLP_inference(device, model_version, tokenizer_version, batch,
 def test_LlamaMLP_inference(model_version, tokenizer_version, batch, seq_len, on_weka, pcc):
     # Initialize the device
     device = ttnn.open_device(0)
-    ttnn.experimental.device.SetDefaultDevice(device)
+    ttnn.SetDefaultDevice(device)
 
     run_test_LlamaMLP_inference(device, model_version, tokenizer_version, batch, seq_len, on_weka, pcc)
-    ttnn.close_device(device)
+    ttnn.CloseDevice(device)
```

Comment:
do we need to change this from close_device ?

Reply:
No. will revert it. Thankyou